betatest/urls.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+URL Patterns for the `betatest` app
+"""
+
+# Imports #####################################################################
+
+from django.conf.urls import url
+
+from betatest import views
+
+
+# URL Patterns ################################################################
+
+app_name = 'beta'
+urlpatterns = [
+    url(r'^register/$', views.BetaTestApplicationView.as_view(), name='register'),
+    url(r'^success/$', views.success, name='success'),
+    url(r'^api/register/validate/$', views.validate_registration, name='validate'),
```

Comment:
@omarkhan Thanks for the details. DRF Docs looks neat, it's a good idea imho - do you want to add it to the uservoice, so we consider it to replace the current approach in the future?